docs/configuration.v2alpha1.md

```diff
@@ -1,18 +1,28 @@
 # Configuration
 
+This page describes configuration options for Datadog Operator v1.0.0, which uses `DatadogAgent/v2alpha1`. For earlier versions of the Datadog Operator, see [configuration options for `DatadogAgent/v1alpha1`][7] or [migrate to `v2alpha1`][8].
```

Comment:
This whole file is generated. This section should be updated in the `header.markdown`
https://github.com/DataDog/datadog-operator/blob/main/hack/generate-docs/header.markdown